frontend/src/pages/projects/utils.ts

```diff
@@ -5,8 +5,10 @@ import { NotebookState } from './notebook/types';
 export const getNotebookStatusPriority = (notebookState: NotebookState): number =>
   notebookState.isRunning ? 1 : notebookState.isStarting ? 2 : 3;
 
-export const getPvcTotalSize = (pvc: PersistentVolumeClaimKind): string =>
-  pvc.status?.capacity?.storage || pvc.spec.resources.requests.storage;
+export const getPvcTotalSize = (pvc: PersistentVolumeClaimKind): string => {
+  const storage = pvc.status?.capacity?.storage || pvc.spec.resources.requests.storage;
+  return /(Gi|Mi)$/.test(storage) ? `${storage}B` : storage;
+};
```

Comment:
@jenny-s51 why doesn't this use the utility function?

Reply:
Thank you @christianvogt - applying the utility function here now